mava/utils/training.py

```diff
@@ -56,3 +59,74 @@ def make_learning_rate(init_lr: float, config: DictConfig) -> Union[float, Calla
         return make_learning_rate_schedule(init_lr, config)
     else:
         return init_lr
+
+
+def select_action_ppo(
+    actor_output: Tuple[Array, Array],
+    key: PRNGKey,
+    env_name: str,
+) -> Tuple[Array, Array, Array]:
+    """Select action for the given actor output."""
+
+    actor_mean, actor_log_std = actor_output
+    actor_policy = distrax.MultivariateNormalDiag(actor_mean, jnp.exp(actor_log_std))
+
+    raw_action, log_prob = actor_policy.sample_and_log_prob(seed=key)
+    action, log_prob = transform_actions_log(env_name, raw_action, log_prob)
+
+    return raw_action, action, log_prob
+
+
+def get_logprob_entropy(
+    actor_output: Tuple[Array, Array],
+    traj_action: Array,
+    env_name: str,
+) -> Tuple[Array, Array]:
+    """Get the log probability and entropy of a given actor output and traj_batch action."""
+
+    actor_mean, actor_log_std = actor_output
+    actor_policy = distrax.MultivariateNormalDiag(actor_mean, jnp.exp(actor_log_std))
+
+    log_prob = actor_policy.log_prob(traj_action)
+    entropy = actor_policy.entropy().mean()
+
+    _, log_prob = transform_actions_log(env_name, traj_action, log_prob)
+    return log_prob, entropy
```

Comment:
naming and separating 

```suggestion
def get_logprob_entropy(
    mean: Array,
    log_std: Array,
    action: Array,
    env_name: str,
) -> Tuple[Array, Array]:
    """Get the log probability and entropy of a given mean, log_std and action."""

    actor_policy = distrax.MultivariateNormalDiag(mean, log_std)

    log_prob = actor_policy.log_prob(action)
    entropy = actor_policy.entropy().mean()

    _, log_prob = transform_actions_log(env_name, action, log_prob)
    return log_prob, entropy
```